File: PgAce.App/Forms/SqlExplorerPane.vb
Imports System.Windows.Forms
Imports WeifenLuo.WinFormsUI.Docking

Namespace PgAce.App.Forms
    Public Class SqlExplorerPane
        Inherits DockContent

        Public Sub New()
            Text = "SQL Explorer"
            Dim tree = New TreeView() With {
                .Dock = DockStyle.Fill
            }
            Controls.Add(tree)
        End Sub
    End Class
End Namespace

File: PgAce.App/Forms/SqlExplorerPane.cs
using System.Windows.Forms;
using WeifenLuo.WinFormsUI.Docking;

namespace PgAce.App.Forms
{
    public class SqlExplorerPane : DockContent
    {
        public SqlExplorerPane()
        {
            Text = "SQL Explorer";
            var tree = new TreeView
            {
                Dock = DockStyle.Fill,
            };
            Controls.Add(tree);
        }
    }
}
